tests/stress/stress_test.go

```diff
@@ -42,32 +46,65 @@ func runTest(b *testing.B, c StressTestCase) {
 	terraform.Apply()
 	defer terraform.Destroy()
 
-	methodsToBenchmark := map[string]func(){
-		"ListDeploys":       func() { layer0.ListDeploys() },
-		"ListEnvironments":  func() { layer0.ListEnvironments() },
-		"ListLoadBalancers": func() { layer0.ListLoadBalancers() },
-		"ListServices":      func() { layer0.ListServices() },
-		"ListTasks":         func() { layer0.ListTasks() },
-	}
+	methodsToBenchmark := map[string]func(){}
 
 	if c.NumDeploys > 0 {
 		deployIDs := strings.Split(terraform.Output("deploy_ids"), ",\n")
 		methodsToBenchmark["GetDeploy"] = func() { layer0.GetDeploy(deployIDs[0]) }
+		methodsToBenchmark["ListDeploys"] = func() { layer0.ListDeploys() }
 	}
 
 	if c.NumEnvironments > 0 {
 		environmentIDs := strings.Split(terraform.Output("environment_ids"), ",\n")
 		methodsToBenchmark["GetEnvironment"] = func() { layer0.GetEnvironment(environmentIDs[0]) }
+		methodsToBenchmark["ListEnvironments"] = func() { layer0.ListEnvironments() }
 	}
 
 	if c.NumLoadBalancers > 0 {
 		loadBalancerIDs := strings.Split(terraform.Output("load_balancer_ids"), ",\n")
 		methodsToBenchmark["GetLoadBalancer"] = func() { layer0.GetLoadBalancer(loadBalancerIDs[0]) }
+		methodsToBenchmark["ListLoadBalancers"] = func() { layer0.ListLoadBalancers() }
 	}
 
 	if c.NumServices > 0 {
 		serviceIDs := strings.Split(terraform.Output("service_ids"), ",\n")
 		methodsToBenchmark["GetService"] = func() { layer0.GetService(serviceIDs[0]) }
+		methodsToBenchmark["ListServices"] = func() { layer0.ListServices() }
+	}
+
+	if c.NumTasks > 0 {
+		methodsToBenchmark["ListTasks"] = func() { layer0.ListTasks() }
+
+		deployIDs := strings.Split(terraform.Output("deploy_ids"), ",\n")
+		environmentIDs := strings.Split(terraform.Output("environment_ids"), ",\n")
+
+		tasksCreated := 0
+		for copies := c.NumTasks / 2; tasksCreated < c.NumTasks; copies = copies / 2 {
```

Comment:
This `copies` logic is a little odd to me, especially the `copies = copies / 2` . 
I think there's a more simple/clear way to do what you want here. 

However, I don't think we need to mess with copies at all in this test. 
Reason being, unlike `Deploys` where the distribution among the family makes a difference in underlying architecture/api calls, 
the number of copies for a task is more-or-less an alias for running `TaskCreate()` N times.

Reply:
The `copies` logic is a way to sort-of recreate the different batches of Tasks that you had in that map in the old test (i.e. simulating multiple users submitting Tasks), but without hardcoding batch sizes or knowing how many total Tasks are to be created:
```
taskNameCopies := map[string]int{
		"TaskA": 50,
		"TaskB": 25,
		"TaskC": 10,
		"TaskD": 5,
		"TaskE": 5,
		"TaskF": 1,
		"TaskG": 1,
		"TaskH": 1,
		"TaskI": 1,
		"TaskJ": 1,
}
```

For example, given 100 tests, this loop would make batch submissions of 50, 25, 12, 6, 3, 1, 1, 1, 1.

Reply:
I think it's quite a bit of complexity for little-to-no benefit, since, under the covers, it's essentially doing `RunTask()` N times. 

Also fwiw,  I'm going to push for removing `copies` in the api refactor

Reply:
Cool, I'll gut it here.

✅